packages/twenty-server/src/workspace/messaging/query-hooks/message/message-channel-message-assocatiation-find-many.post-query-hook.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { Injectable } from '@nestjs/common';
+
+import { IConnection } from 'src/utils/pagination/interfaces/connection.interface';
+import { IEdge } from 'src/utils/pagination/interfaces/edge.interface';
+import { WorkspacePostQueryHook } from 'src/workspace/workspace-query-runner/workspace-query-hook/interfaces/workspace-post-query-hook.interface';
+
+import { EMPTY_PAGE } from 'src/workspace/workspace-query-runner/workspace-query-hook/types/workspace-query-hook.type';
+
+@Injectable()
+export class MessageChannelMessageAssociationFindManyPostQueryHook
+  implements WorkspacePostQueryHook
+{
+  async execute<Record>(
```

Comment:
We don't want to return results from this endpoint. This is mostly for testing, in practice you want to return earlier, using pre-hook and throwing an exception if it's called.
However this is only working for people directly calling messageChannelMessageAssociations endpoint, this won't be called for nested types inside other more accessible types which means we can't fully trust this feature for security logic